docs/book/src/user/quick-start.md

```diff
@@ -928,6 +928,7 @@ are also a few optional tunables if you'd like to change the OS or CIDRs used.
 export PROJECT_ID="2b59569f-10d1-49a6-a000-c2fb95a959a1"
 # The facility where you want your cluster to be provisioned
 export FACILITY="da11"
+export METRO="da"
```

Comment:
Could you add  a comment explaining what this env variable is - like in the others in the list.

I think you also need to sign the CLA to pass that check.

Reply:
METRO ENV variable can help to take advantage of automated, interconnected bare metal across our global metros.

Reply:
Can you add this information inline in the markdown - just a simple comment stating what the env variable is used for is enough.